go/libkb/chain_link.go

```diff
@@ -106,6 +110,10 @@ type ChainLinkUnpacked struct {
 	stubbed                            bool
 	firstAppearedMerkleSeqnoUnverified keybase1.Seqno
 	payloadHash                        []byte
+	sigDropped                         bool
+	hasRevocations                     bool
```

Comment:
do you still need this field?

Reply:
not necessarily, but it will be a slight optimization for verifyPayloadV2 that we already know this.